app/gateway/changelog.md

```diff
@@ -5378,6 +5378,21 @@ openid-connect
 * Bumped `lodash` for Dev Portal from 4.17.11 to 4.17.21
 * Bumped `lodash` for Kong Manager from 4.17.15 to 4.17.21
 
+## 2.8.4.12
+**Release Date** 2024/07/29
+
+### Breaking changes and deprecations
+
+* Debian 10 and RHEL 7 reached their End of Life (EOL) dates on June 30, 2024. 
+As of this patch, Kong is not building Kong Gateway 2.8.x installation packages or Docker images for these operating systems.
```

Comment:
Is there another PR where those are removed from this table? https://deploy-preview-7693--kongdocs.netlify.app/gateway/3.7.x/support-policy/

Reply:
They don't need to be removed, they have an EOL in the final column for those versions that's already set to June 2024. Existing packages are still supported, there's just no new ones as of this patch.